ftp.go

```diff
@@ -320,7 +314,7 @@ func parseListLine(line string) (*Entry, error) {
 					e.Type = EntryTypeFile
 				}
 			case "size":
-				e.Size, _ = strconv.ParseUint(value, 0, 64)
+				e.setSize(value)
```

Comment:
But the result seems to be the same. I do not really understand the difference, do you?